src/pages/Travel/TravelTerms.tsx

```diff
@@ -0,0 +1,111 @@
+import React, {useCallback, useEffect, useState} from 'react';
+import {View} from 'react-native';
+import {ScrollView} from 'react-native-gesture-handler';
+import FullPageNotFoundView from '@components/BlockingViews/FullPageNotFoundView';
+import CheckboxWithLabel from '@components/CheckboxWithLabel';
+import FormAlertWithSubmitButton from '@components/FormAlertWithSubmitButton';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import SafeAreaConsumer from '@components/SafeAreaConsumer';
+import ScreenWrapper from '@components/ScreenWrapper';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import useLocalize from '@hooks/useLocalize';
+import usePermissions from '@hooks/usePermissions';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import * as Travel from '@userActions/Travel';
+import CONST from '@src/CONST';
+
+function TravelTerms() {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const {canUseSpotnanaTravel} = usePermissions();
+    const [hasAcceptedTravelTerms, setHasAcceptedTravelTerms] = useState(false);
+    const [error, setError] = useState(false);
+
+    const errorMessage = error ? 'travel.termsAndConditions.error' : '';
+
+    const toggleTravelTerms = () => {
+        setHasAcceptedTravelTerms(!hasAcceptedTravelTerms);
+    };
+
+    useEffect(() => {
+        if (!hasAcceptedTravelTerms) {
+            return;
+        }
+
+        setError(false);
+    }, [hasAcceptedTravelTerms]);
+
+    const AgreeToTheLabel = useCallback(
```

Comment:
Could we have used `useMemo` insetad of `useCallback` here? I am not fully sure why we need to use useCallback in this case